### ISSUE TITLE
Jenkins slave exceeded quota on import

### ISSUE THREAD

**Issue by burrsutter** [openshiftio/openshift.io#550]
During Jenkins builds.
Imported all 3 of
https://github.com/burrsutter/vertx-eventbus
https://github.com/burrsutter/vertx-paint
https://github.com/burrsutter/vertx-wiki2

```
Aug 10, 2017 7:03:30 PM io.fabric8.jenkins.openshiftsync.BuildSyncRunListener onCompleted
--
  | INFO: onCompleted job/burrzinga/job/vertx-paint/job/master/1/
  | Aug 10, 2017 7:03:31 PM io.fabric8.jenkins.openshiftsync.BuildSyncRunListener onFinalized
  | INFO: onFinalized job/burrzinga/job/vertx-paint/job/master/1/
  | Aug 10, 2017 7:03:39 PM hudson.slaves.NodeProvisioner$2 run
  | WARNING: Unexpected exception encountered while provisioning agent Kubernetes Pod Template
  | io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://172.30.0.1/api/v1/namespaces/burrzinga-jenkins/pods. Message: Forbidden!Configured service account doesn't have access. Service account may have been revoked. pods "jenkins-slave-zzh20-4st4w" is forbidden: exceeded quota: compute-resources-timebound, requested: limits.cpu=2500m,limits.memory=1280Mi, used: limits.cpu=2500m,limits.memory=1280Mi, limited: limits.cpu=4,limits.memory=2Gi.
  | at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:470)
  | at io.fabric8.kubernetes.client.dsl.base.OperationSupport.assertResponseCode(OperationSupport.java:407)
  | at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
  | at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:343)
  | at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:226)
  | at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:754)
  | at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:344)
  | at org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud$ProvisioningCallback.call(KubernetesCloud.java:687)
  | at org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud$ProvisioningCallback.call(KubernetesCloud.java:623)
  | at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
  | at java.util.concurrent.FutureTask.run(FutureTask.java:266)
  | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  | at java.lang.Thread.run(Thread.java:748)

```

**Comment by rawlingsj** [comment]
Where are you seeing this? Is it from the Jenkins pod logs and what impact does this have? 

I would have though this is a non issue as if you import three projects that will trigger three pipelines, but due to quotas only one build pod can run at a time. I would expect all build pods to eventually complete as the Jenkins builds will wait until kubernetes can schedule its specific build pod.

**Comment by rawlingsj** [comment]
As above I don't believe this is an issue.  The log is a warning that there's no quota to start build pods for two out of the three repos imported, this is because there's already one running for the first application.  Once the first build finishes Kubernetes then automatically schedules the second build pod, the Jenkins build then continues for the second imported repo and so on until all jobs complete.

TL;DR everything works as expected based on the restricted quotas in the Jenkins namespace.

I'm going to close this based on the explanation above.
